doc/source/examples/multiphysics/laser-heating/laser-heating.rst

```diff
@@ -0,0 +1,231 @@
+==========================
+Laser heating
+==========================
+
+This example simulates a three-dimensional solid block heated with a laser beam. 
+
+----------------------------------
+Features
+----------------------------------
+- Solver: ``gls_navier_stokes_3d`` 
+- Laser heat source
+- Convection-radiation heat transfer boundary condition
+- Unsteady problem handled by an adaptive BDF1 time-stepping scheme 
```

Comment:
Would be cool to use BDF2 here instead and use this as an opportunity to check it works well for temperature also.